app/src/test/java/com/cmput301w23t09/qrhunter/TestMapFragment.kt

```diff
@@ -0,0 +1,5 @@
+package com.cmput301w23t09.qrhunter
+
+public class TestMapFragment {
```

Comment:
Please remove this or implement map tests.
When implementing map tests, please use the .java file extension instead of Kotlin.